src/viam/sdk/registry/registry.cpp

```diff
@@ -82,8 +92,12 @@ Status ModelRegistration::create_status(std::shared_ptr<ResourceBase> resource)
     return status;
 }
 
-std::unordered_map<std::string, std::shared_ptr<ModelRegistration>> Registry::resources;
-std::unordered_map<Subtype, std::shared_ptr<ResourceSubtype>> Registry::subtypes;
+const google::protobuf::ServiceDescriptor* ResourceSubtype::service_descriptor() {
```

Comment:
On the other hand, this const return *is* meaningful because the const-ness is about the object pointed to.